_locales/pt_PT/messages.json

```diff
@@ -1,20 +1,27 @@
 {
-    "appNameNewTab": {
-        "message": "DuckieTV - Modo 'New Tab' "
-    },
-    "appShortNameNewTab": {
-        "message": "DuckieTV "
-    },
-    "appDescNewTab": {
-        "message": "Tudo o que você precisa saber quando você vem favorito Series TV internacional em "
-    },
-    "appNameBrowserAction": {
-        "message": "DuckieTV - Modo Botão Navegador '"
-    },
-    "appShortNameBrowserAction": {
-        "message": "DuckieTV "
-    },
-    "appDescBrowserAction": {
-        "message": "Tudo o que você precisa saber quando você vem favorito Series TV internacional em"
-    }
-}
\ No newline at end of file
+	"appNameNewTab": {
+		"message": "DuckieTV - Modo 'Nova Aba' - Nightly",
```

Comment:
I have only just now realised that you appear to be wanting to make a change for just the **nightly** releases.
I wished I spotted this sooner, I could have saved you a lot of wasted effort.
We don't keep separate source code for the nightlys, and another for the published main release, they are one and the same.
The only difference between the packages is the cosmetic alteration done by the release bot, to change *version 1.1.6* into a  *version timestamp*. 

So if I accept your commits then we will end up with the word *Nightly* in the published main release.
Sorry :-(

but if you have any other corrections to phrases or spelling then i will happily accept those.